--- a/src/opentelemetry/test/test_base.py
+++ b/src/opentelemetry/test/test_base.py
@@ -70,10 +70,6 @@
             self, self.memory_exporter.get_finished_spans()
         )
 
-    def assertEqualSpanInstrumentationInfo(self, span, module):
-        self.assertEqual(span.instrumentation_info.name, module.__name__)
-        self.assertEqual(span.instrumentation_info.version, module.__version__)
-
     def assertSpanHasAttributes(self, span, attributes):
         for key, val in attributes.items():
             self.assertIn(key, span.attributes)
